<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Integration Time</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxInt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>8.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSetInt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fiber</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton100um">
         <property name="text">
          <string>100 µm</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton1000um">
         <property name="text">
          <string>1000 µm</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Others</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBoxSlit">
         <property name="text">
          <string>Slit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDark">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Dark
Correction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxNonlinear">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Nonlinearity
Correction</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QWidget" name="mplwindow" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplvl"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAbsorbance"/>
    <addaction name="actionPlot"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="actionOpenDev"/>
    <addaction name="actionSpectrum"/>
    <addaction name="actionMultiRun"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDev"/>
   <addaction name="actionSpectrum"/>
   <addaction name="actionMultiRun"/>
   <addaction name="separator"/>
   <addaction name="actionAbsorbance"/>
   <addaction name="actionPlot"/>
   <addaction name="actionSave"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/Save as.png</normaloff>:/icon/icon/Save as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/Self-Destruct Button.png</normaloff>:/icon/icon/Self-Destruct Button.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/plot.png</normaloff>:/icon/icon/plot.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Plot...</string>
   </property>
   <property name="toolTip">
    <string>Plot...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOpenDev">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/Disconnected.png</normaloff>
     <normalon>:/icon/icon/Connected.png</normalon>:/icon/icon/Disconnected.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Device</string>
   </property>
   <property name="toolTip">
    <string>Open Device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSpectrum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/Rainbow.png</normaloff>:/icon/icon/Rainbow.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Spectrum</string>
   </property>
   <property name="toolTip">
    <string>Acquire Spectrum</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbsorbance">
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/absorb.png</normaloff>:/icon/icon/absorb.png</iconset>
   </property>
   <property name="text">
    <string>Absorbance</string>
   </property>
   <property name="toolTip">
    <string>Calculate Absorbance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionMultiRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="spectrometer.qrc">
     <normaloff>:/icon/icon/Ice Cream Cone.png</normaloff>:/icon/icon/Ice Cream Cone.png</iconset>
   </property>
   <property name="text">
    <string>Multiple Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="spectrometer.qrc"/>
 </resources>
 <connections/>
</ui>
